internal/lang/marks/marks.go

```diff
@@ -45,3 +50,123 @@ const Sensitive = valueMark("Sensitive")
 // another value's type. This is part of the implementation of the console-only
 // `type` function.
 const TypeType = valueMark("TypeType")
+
+type DeprecationCause struct {
+	By      addrs.Referenceable
+	Message string
+}
+
+type deprecationMark struct {
+	Cause DeprecationCause
+}
+
+func (m deprecationMark) GoString() string {
+	return "marks." + string("Deprecated")
```

Comment:
Why keep the + string() here?

Reply:
That's more of a typo. I was initially planning to put more info in there and forgot to change back. Removed in [66c6a0e](https://github.com/opentofu/opentofu/pull/2633/commits/66c6a0e16be6c61754e965e73a330e181fa8be5c)